resource/hlapi/bindings/c++/reapi_cli_impl.hpp

```diff
@@ -97,7 +99,29 @@ int reapi_cli_t::match_allocate (void *h, bool orelse_reserve,
 
         R = o.str ();
 
-        if ( (rc = gettimeofday (&end_time, NULL)) < 0) {
+
+        reserved = (at != 0)? true : false; 
+        st = (reserved)? 
+                    job_lifecycle_t::RESERVED : job_lifecycle_t::ALLOCATED; 
+        if (reserved) 
+            rq->set_reservation (jobid); 
+        else
+            rq->set_allocation (jobid);
+ 
+        job_info = std::make_shared<job_info_t> (jobid, st, at, "", "", ov); 
+        if (job_info == NULL) {
+            errno = ENOMEM; 
+            m_err_msg += __FUNCTION__; 
+            m_err_msg += ": ERROR: can't allocate memory: " 
+                         + std::string (strerror (errno))+ "\n"; 
+            rc = -1; 
+            goto out; 
+        } 
+ 
+        rq->set_job (jobid, job_info); 
+        rq->incr_job_counter ();
+                         
+        if ((rc = gettimeofday (&end_time, NULL)) < 0) {
```

Comment:
Nit: Fluxion uses a space between double open parentheses for conditionals.

```suggestion
        if ( (rc = gettimeofday (&end_time, NULL)) < 0) {
```